src/pymatgen/analysis/local_env.py

```diff
@@ -197,7 +196,7 @@ def _get_valences(self):
         return valences
 
 
-on_disorder_options = Literal["take_majority_strict", "take_majority_drop", "take_max_species", "error"]
+on_disorder_options: TypeAlias = Literal["take_majority_strict", "take_majority_drop", "take_max_species", "error"]  # noqa: PYI042
```

Comment:
this type alias name doesn't conform with camel case, but wouldn't change it for now to avoid potential breakage